### PR TITLE
Update docker images to newest Ubuntu 20.04, split up services to sep…

### PR DESCRIPTION
**Pull request by h44z**
This pull request improves the docker integration of daloRADIUS.

Changes:
 - update docker images to newest Ubuntu 20.04
 - split up services to three separate containers (mysql, freeradius, daloradius)
 - provide a docker-compose.yml file
 - ensure that service restart is safe (do not overwrite mysql entries once they got initialized)
 
The docker-compose file contains all environment variables that can be used currently, optional variables can be removed from docker-compose if desired.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,62 +8,70 @@
...
 	Version="1.0"
 
-# silence package installations to that debpkg doesn't prompt for mysql
-# passwords and other input
-ENV DEBIAN_FRONTEND "noninteractive"
-ENV mysql_pass ""
+ENV DEBIAN_FRONTEND noninteractive
+
+# default timezone
+ENV TZ Europe/Vienna
```
[the remark this answers is not in the document]
Comment (h44z) [reply Dockerfile:22]: I am not really sure which timezone would make the most sense a default value. Do you have any preferences/suggestions?

File: Dockerfile [hunk 2]
```diff
@@ -8,62 +8,70 @@
 # Run the container:
 # 1. docker run -p 80:80 -d lirantal/daloradius
 
-FROM ubuntu:16.04
+FROM ubuntu:20.04
```
[the remark this answers is not in the document]
Comment (h44z) [reply Dockerfile:11]: yes, assuming that the code works under php 7.4. But as far as I was able to test it, I found no problems.

[the remark this answers is not in the document]
Comment (lirantal) [reply Dockerfile:11]: Cool thanks. Will try to build this locally tomorrow and run a quick sanity test too!

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (lirantal) [reply Dockerfile:22]: Perhaps just UTC as a default?

**Comment by lirantal** [comment]
@h44z first of all appreciate the work on this 🙏 

I ran a local build of the `Dockerfile`, and then `docker-compose up`.
While the app is starting up, it seems the DB connection isn't well set up.

This is what I see in the docker-compose logs:

```
radius          | Listening on auth address * port 1812 bound to server default
radius          | Listening on acct address * port 1813 bound to server default
radius          | Listening on auth address :: port 1812 bound to server default
radius          | Listening on acct address :: port 1813 bound to server default
radius          | Listening on auth address 127.0.0.1 port 18120 bound to server inner-tunnel
radius          | Listening on status address 127.0.0.1 port 18121 bound to server status
radius          | Listening on proxy address * port 49355
radius          | Listening on proxy address :: port 54143
radius          | Ready to process requests
radius-mysql    | 2020-12-04 17:14:39 18 [Warning] Access denied for user 'root'@'172.20.0.4' (using password: NO)
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | sed: can't read /var/www/html/library/daloradius.conf.php: No such file or directory
radius-web      | daloRADIUS initialization completed.
radius-web      | Database initialization for daloRADIUS completed.
radius-web      | AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.20.0.4. Set the 'ServerName' directive globally to suppress this message
radius-mysql    | 2020-12-04 17:24:38 17 [Warning] Aborted connection 17 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
radius-mysql    | 2020-12-04 17:24:38 14 [Warning] Aborted connection 14 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
radius-mysql    | 2020-12-04 17:24:38 16 [Warning] Aborted connection 16 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
radius-mysql    | 2020-12-04 17:24:38 12 [Warning] Aborted connection 12 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
radius-mysql    | 2020-12-04 17:24:38 15 [Warning] Aborted connection 15 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
radius-mysql    | 2020-12-04 17:24:38 13 [Warning] Aborted connection 13 to db: 'radius' user: 'radius' host: '172.20.0.3' (Got timeout reading communication packets)
```

**Comment by lirantal** [comment]
Looks like the daloradius container doesn't have a `daloradius.conf.php` file, just the sample one.

**Comment by lirantal** [comment]
@h44z okie, fixed this by copying the sample file to an actual one if it doesn't exist.
looks great to me so I'll go ahead and merge this in.

@h44z amazing job pulling all of this in. thank you so much!

**Pull request merged**